packages/neuron-wallet/src/block-sync-renderer/tx-status-listener.ts

```diff
@@ -61,7 +61,7 @@ const trackingStatus = async () => {
     const blake160s = await FailedTransaction.updateFailedTxs(failedTxs.map(tx => tx.hash))
     const prefix = NetworksService.getInstance().isMainnet() ? AddressPrefix.Mainnet : AddressPrefix.Testnet
     const usedAddresses = blake160s.map(blake160 => AddressGenerator.toShortByBlake160(blake160, prefix))
-    await WalletService.updateUsedAddresses(usedAddresses)
+    await WalletService.updateUsedAddresses(usedAddresses, blake160s)
```

Comment:
Are `block160s` the ACP related lock hashes? If so, it may be better to include the ACP in terms of variable naming. For example: `anyoneCanPayBlake160s`